WSLBlobNFS/wsl2_linux_script.sh

```diff
@@ -171,8 +411,8 @@ function export_via_samba ()
     # testparm
```

Comment:
whys is this commented?
if it's a todo, mark it as such

Reply:
restarting will do the same check.. will handle after restarting